src/System.Linq.Expressions/tests/HelperTypes.cs

```diff
@@ -391,6 +395,68 @@ public static Type BoolEnumType
                 return _boolEnumType;
             }
         }
+        
+        public static Type FloatEnumType
```

Comment:
Float enums? And people call mutable structs evil! Again though, if the spec allows such an enum, IMO expressions should support them.